api/v2/models/receiver.go

```diff
@@ -31,6 +33,14 @@ import (
 // swagger:model receiver
 type Receiver struct {
 
+	// active
+	// Required: true
+	Active *bool `json:"active"`
+
+	// integrations
+	// Required: true
```

Comment:
In that case the `integrations` field would be an empty array.

Reply:
yes, but it can be a null too.

Reply:
I think that's exactly what we're trying to avoid here. That way clients always get an array instead of having to check whether the value is null first, running the risk of having runtime errors.

Reply:
I get it. Fine.